### DISCUSSION TITLE
Command+. on MacOS

### DISCUSSION THREAD

**Discussion by theodopolopodous** [kovidgoyal/kitty#6128] [General]
Hello, thank you for the cool project

I am a new user, and am noticing that Command+. on MacOS types "6;9u"

Weird thing is, 10% of the time if correctly brings up the Settings, and there's no discernible difference why it works occasionally.

**Comment by kovidgoyal** [comment]
There is no default cmd+. The shortcut is cmd+, and it works every time
for me on my mac.

**Comment by theodopolopodous** [comment]
Right you are!  Apologies for the atrocious question, my eyes completely misread that character
